packages/dev-approuter/README.md

```diff
@@ -0,0 +1,179 @@
+
+```yaml
+specVersion: "3.0"
+metadata:
+  name: ui5-app  # default mount path would be /ui5-app 
+type: application
+customConfiguration:
+  mountPath: /my-custom-mount-path # overwrites the default mount path
+```
+
+The above configuration result in the UI5 app being available at `http://localhost:5000/my-custom-mount-path`.
+
+### SAP CAP apps
+
+SAP CAP apps are started on a separate port (4004 by default, can be overwritten via `CAP_PORT`). The `dev-approuter` automatically creates a route and destination behind the scenes, so that you (and your UI5 apps) can reach your SAP CAP services directly at their service path, but through the `dev-approuter`, e.g. `http://localhost:5000/my-cap-service`.
+
+There is no need manually create a destination for you SAP CAP app, unless you want to overwrite the [default destination configuration](./lib/helpers.js#L81-L85). In this case, create a new destination in a `default-env.json`. The destination's name has to match the module name as declared in the approuter's `devDependencies`:
+
+```json
+{
+    "CAP_PORT": 4005,
+    "destinations": [
+        {
+            "Name": "my-cap-app",
+            "Authentication": "NoAuthentication",
+            "ProxyType": "Internet",
+            "Type": "HTTP",
+            "URL": "http://localhost:4005",
+            "forwardAuthToken": true
+        }
+    ],
+    ...
+}
+```
+
+## The `xs-dev.json` file
+
+The `dev-approuter` introduces the concept of an `xs-dev.json` file, which works like a regular [`xs-app.json`](https://help.sap.com/docs/btp/sap-business-technology-platform/routing-configuration-file) file, but is used by the `dev-approuter` exclusively (meaning it's ignored by the SAP Approuter). The idea behind this concept is to safely separate dev time configuration from productive code.
+
+The `xs-dev.json` follows the same logic and syntax as the [`xs-app.json`](https://help.sap.com/docs/btp/sap-business-technology-platform/routing-configuration-file) file, but has one additional key feature: You can add a `dependency` to a `route`, which links it to a UI5 or SAP CAP app.
+
+Look at the following example `xs-dev.json` that defines different `authenticationType`s for different UI5 apps:
+
+```json
+{
+    "welcomeFile": "index.html",
+    "authenticationMethod": "route",
+    "routes": [
+        {
+        "dependency": "my-ui5-app1",
+        "authenticationType": "none"
+        },
+        {
+        "dependency": "my-ui5-app2",
+        "authenticationType": "xsuaa"
+        }
+    ]
+}
+```
+
+Behind the scenes, the `dev-approuter` will resolve these "dependency routes" by adding the `source`, `target`, and `destination` properties to them. Be aware that exactly these properties might get overwritten by the `dev-approuter` in case you use them together with `dependency`.
+
+## Using the `dev-approuter` and SAP Approuter simultaneously
```

Comment:
Again SAP Approuter => alternative short name is camel cased => SAP AppRouter - but in this case I would prefer the long name... Or use the technical name: `@sap/approuter`

Reply:
Ok